TCAT/Controllers/RouteOptionsViewController.swift

```diff
@@ -33,11 +33,11 @@ enum RequestAction {
 
 class RouteOptionsViewController: UIViewController {
 
-    var datePickerOverlay: UIView!
-    var datePickerView: DatePickerView!
-    var routeResults: UITableView!
-    var routeSelection: RouteSelectionView!
-    var searchBarView: SearchBarView?
+    let datePickerOverlay = UIView()
+    let datePickerView = DatePickerView()
+    let routeResults = UITableView(frame: .zero, style: .grouped)
+    let routeSelection = RouteSelectionView()
+    let searchBarView = SearchBarView()
```

Comment:
make `private`

Reply:
All of these end up getting accessed in the extensions file. I'll try to reorganize these when tackling #251 and #252